server/auth/auth.go

```diff
@@ -671,6 +671,27 @@ func (auth *AuthManager) Notify(acctID account.AccountID, msg *msgjson.Message,
 // Penalize signals that a user has broken a rule of community conduct, and that
 // their account should be penalized.
 func (auth *AuthManager) Penalize(user account.AccountID, rule account.Rule) error {
+	// Notify user of penalty.
+	details := "You may no longer trade. Leave your client running to complete outstanding trades."
```

Comment:
How about "Ordering has been suspended for this account. Contact the exchange operator to reinstate privileges."

Do we need custom messaging for when `auth.anarchy == true`?

Reply:
I asked for it to be moved up even in that case.  The use case for anarchy is testing, and I'd like the testers to know when they would be suspending their accounts.

Reply:
Sorry. Just meant to ask if the message itself should be customized to explain that they actually can continue to trade, because anarchy.